core/iwasm/libraries/lib-socket/test/build.sh

```diff
@@ -0,0 +1,13 @@
+
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+/opt/wasi-sdk/bin/clang \
```

Comment:
I'd recommend taking compiler from env variable (similarly to https://github.com/WebAssembly/wasi-testsuite/blob/main/tests/c/build.sh#L4) and fall-back to `/opt/was-sdk/bin/clang` if not provided - not everybody install wasi-sdk in that location so there should be a way to override it.